README.md

```diff
@@ -71,6 +71,12 @@ You've successfully run and modified your React Native App. :partying_face:
 
 If you can't get this to work, see the [Troubleshooting](https://reactnative.dev/docs/troubleshooting) page.
 
+# Building your App
```

Comment:
If yo uuse the command-line interface, I think it would be beneficial to also have a step-by-step recipe here for how to do it in this particular project. Something like this:

```sh
cd android
gradle...
...and so on
```